check-lockfile.sh

```diff
@@ -0,0 +1,8 @@
+echo "checking Yarn lockfile for changes"
```

Comment:
Totally should've thought about this earlier but I think some context would be useful here. Something like…

> Check to see whether the yarn lockfile was changed when Travis automatically ran yarn. We can't use yarn check --integrity because the lock file will already have been updated (but not committed), and so the check will always pass.

?